easybuild/easyconfigs/p/PyTorch/PyTorch-1.10.1-CrayGNU-21.09.eb

```
@@ -0,0 +1,85 @@
+# contributed by Luca Marsella (CSCS)
+easyblock = 'PythonBundle'
+
+name = 'PyTorch'
+version = '1.10.1'
+_ptcudaver = 'cu113'
+
+homepage = 'https://pytorch.org'
+description = """Tensors and Dynamic neural networks in Python with strong GPU acceleration.
+PyTorch is a deep learning framework that puts Python first."""
+
+toolchain = {'name': 'CrayGNU', 'version': '21.09'}
+toolchainopts = {'pic': True, 'verbose': False}
+
+builddependencies = [
+    ('cudatoolkit/11.2.0_3.39-2.1__gf93aa1c', EXTERNAL_MODULE),
+    ('libffi', '3.4.2'),
+    ('wheel', '0.37.0')
+]
+
+dependencies = [
+    ('Boost', '1.77.0', '-python%(pymajver)s'),
+    ('cray-python', EXTERNAL_MODULE)
+]
+
+preinstallopts = ' export HOROVOD_BUILD_CUDA_CC_LIST=60; '
+preinstallopts += ' export HOROVOD_CUDA_HOME=/usr/local/cuda-11.2; '
```

```suggestion
preinstallopts += ' export HOROVOD_CUDA_HOME=$CUDATOOLKIT_HOME; '
```
however, this wasn't necessary before, so we shouldn't need to define it?

---

Thanks @henrique, I fixed it